semanticdb/metacp/src/main/scala/scala/meta/internal/metacp/Main.scala

```diff
@@ -210,11 +215,16 @@ object Main {
   }
 
   private def sname(sym: Symbol): String = {
-    if (sym.name == "<no symbol>") ""
-    else if (sym.name == "<root>") "_root_"
-    else if (sym.name == "<empty>") "_empty_"
-    else if (sym.name == "<init>") "<init>"
-    else NameTransformer.decode(sym.name)
+    def loop(name: String): String = {
+      val i = name.lastIndexOf("$$")
+      if (i > 0) return loop(name.substring(i + 2))
```

Comment:
can we please reserve `return` for early returns in large method bodies and use if/else in small methods like here?

Reply:
return is inherently harder to reason about

Reply:
Will do.